File: 01_Language/01_Functions/php/gmdate.php
<?php

ini_set("date.timezone", "Asia/Shanghai");
echo date("M d Y H:i:s", mktime(0, 0, 0, 1, 1, 1998)) . "\n";
echo gmdate("M d Y H:i:s", mktime(0, 0, 0, 1, 1, 1998));

echo "--------\n";

echo date("Y-m-d H:i:s") . "\n";
echo gmdate("Y-m-d H:i:s") . "\n";

File: 01_Language/01_Functions/python/gmdate.py
# coding: utf-8

import time


def gmdate(fmt, timestamp=None):
    if timestamp is None:
        t = time.gmtime()
    else:
        t = time.gmtime(timestamp)
    return time.strftime(fmt, t)


if __name__ == '__main__':
    print(gmdate("%Y-%m-%d %H:%M:%S", time.mktime((1998, 1, 1, 0, 0, 0, -1, -1, -1))))

    import date_func

    print(date_func.date("%Y-%m-%d %H:%M:%S"))
    print(gmdate("%Y-%m-%d %H:%M:%S"))
